epub34/annotations/index.html

```diff
@@ -239,7 +239,7 @@ <h3 id="1-1-annotation-object"> Annotation Object </h3>
             </section>
             <section>
 			<h3 id="1-2-creator"> Creator </h3>
-			<p> The <dfn>Creator</dfn> of an annotation is a person or an organization. </p>
+			<p> The <dfn>Creator</dfn> of an annotation is a person, an organization or a software agent. </p>
```

Comment:
```suggestion
			<p> The <dfn>Creator</dfn> of an annotation is a person, an organization, or a software agent. </p>
```

I believe the W3C style is using Oxford commas.